content/blog/2018/06/2018-06-19-new-design-digest.adoc

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: "New design and UX digest"
+tags:
+- core
+- developer
+author: scherler
+---
+
+== Overview
+
+The recent changes to some core pages provide new design and UX and further dropping all external dependencies to prevent
+any possible malicious javascript introduced by third party libraries. https://hackernoon.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5[Why?]
+
+This meant to drop the usage of the jelly layout lib (aka xmlns:l="/lib/layout") and as well the page decorators it
```

Comment:
NIT: I guess I'd wrap the `xmlns...` with backticks. And generally for many if not all the classnames and the likes below like `SimplePageDecorator`, etc.